setup.py

```diff
@@ -18,7 +18,8 @@
     'pillow',
     'appdirs',
     'scooby>=0.5.1',
-    'vtk',
+    'vtk>=8.1.2, <=9.2.0; python_version < "3.10"',
+    'vtk>=8.1.2, <=9.2.0, ==9.2.0rc1; python_version == "3.10"',
```

Comment:
This works if we want to allow any `9.2.x` version.  See https://peps.python.org/pep-0440/#compatible-release

Do we need the other parts for python 3.10 as it isn't supported unless a custom wheel is used?

```suggestion
    'vtk~=9.2.0rc1; python_version == "3.10"',
```

Reply:
I think we should just permit `vtk~=9.2.0rc1` for all python versions. Do you agree?

Reply:
This would install the rc version even on the other versions. I don't think we should do this.  If vtk fully releases 9.2.0 before the next PyVista release, we can use `vtk>=8.1.2, <=9.2.0` without any python versions.